packages/marko-web/components/page/metadata/google-structured-data/content.js

```diff
@@ -66,6 +73,7 @@ module.exports = (node) => {
       '@type': 'PodcastEpisode',
       headline: get(node, 'metadata.title'),
       ...(associatedMedia && { associatedMedia }),
+      ...(get(node, 'transcript') && { transcript: convertHtmlToText(get(node, 'transcript')) }),
```

Comment:
Missing field for `Webinar`s

Reply:
@solocommand This was intentional until we have clearer direction into what type of object a webinar needs to be and what fields need to be add along with the new transcript field.